react/client/src/components/formPages/Step1.tsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
 
+import Button from 'components/Button'
```

Comment:
Updated the buttons for each of the steps just to see how it looks but it'll need a bit more work